crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -833,13 +861,38 @@ type SystemParameters {
 }
 
 type TransactionBlock {
+	"""
+	The transaction block effects provide information such as gas cost and object changes
+	"""
 	effects: TransactionBlockEffects
+	"""
+	Sender of the transaction block
+	"""
 	sender: Address
+	"""
+	The raw transaction block in BCS format
+	"""
 	bcs: Base64
+	"""
+	The gas input of the transaction block
+	"""
 	gasInput: GasInput
 	kind: TransactionBlockKind
+	"""
+	A list of signatures signed by all transaction participants.
+	1. Non-participant signatures must not be present.
+	2. Signature order does not matter
+	"""
 	signatures: [TransactionSignature]
+	"""
+	The digest of a transaction block is represented as a 32-byte array
+	This is returned to callers as a Base58 encoded string
+	"""
 	digest: String!
+	"""
+	Resolves to the expiration epoch of the transaction block
+	Note that this is a value set by the one who executes the transaction block
```

Comment:
> by the one who executes the transaction block?

Can you clarify whether "the one" a validator here, or is it the user that submits the transaction?